UIforETW/UIforETWDlg.cpp

```diff
@@ -2409,3 +2416,59 @@ void CUIforETWDlg::OnActivate(UINT nState, CWnd* pWndOther, BOOL bMinimized)
 	}
 	CDialog::OnActivate(nState, pWndOther, bMinimized);
 }
+
+void CUIforETWDlg::OnMinimize()
+{
+	trayData.cbSize = sizeof(NOTIFYICONDATA);
+	trayData.hWnd = this->m_hWnd;
+	trayData.uID = 1;
+	trayData.uCallbackMessage = WM_TRAY_MESSAGE;
+	trayData.hIcon = this->m_hIcon;
+	
+	wcscpy_s(trayData.szTip, L"UI for ETW");
+	
+
+	trayData.uFlags = NIF_ICON | NIF_MESSAGE | NIF_TIP;
+	
+
+	BOOL bSuccess = FALSE;
+
+	bSuccess = Shell_NotifyIcon(NIM_ADD, &trayData);
+
+	if (!(bSuccess))
+		AfxMessageBox(L"Unable to Set Tary Icon");

```

Comment:
Do we really want to "handle" this error by bringing up a message box? Also, type in Tary/Tray